app/views.py

```diff
@@ -87,8 +87,19 @@ def login():
             return "There was an error.", 500
         # Else success, redirect to index page
         else:
-            session['username'] = username
             session['token'] = token['token']
+            user = getUser(username)
+
+            if type(user) is list:
```

Comment:
@aaroncohen73 Correct. If `TimeSync.get_users()` ever returns a list that check should be done in `get_user(username=...)` and only the user object should be returned. If it only ever returns the user no checks need to be made.